File: javascript/terreno.js
const comprimento = parseFloat(prompt("Digite o comprimento"));
const largura = parseFloat(prompt("Digite a largura"));
const valorM2 = parseFloat(prompt("Digite o valor do metro quadrado"));

const areaTerreno = comprimento * largura;
const precoTerreno = areaTerreno * valorM2;

console.log("Área total do terreno = %.2f", areaTerreno);
console.log("Preço total do terreno = %.2f", precoTerreno);

File: python/terreno.py
largura = float(input("Digite a largura do terreno: "))
comprimento = float(input("Digite o comprimento do terreno: "))
valor_m2 = float(input("Digite o valor_unitario do metro quadrado: "))

area_terreno = largura * comprimento
preco_terreno = valor_m2 * area_terreno

print(f"Area do terreno = {area_terreno:.2f}")
print(f"Preco do terreno = {preco_terreno:.2f}")
